docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_in_amazon_s3.rst

```diff
@@ -3,16 +3,83 @@
 How to configure an Expectation store in Amazon S3
 ==================================================
 
-.. admonition:: Admonition from Mr. Dickens
+By default, newly profiled Expectations are stored in JSON format in the ``expectations/`` subdirectory of your ``great_expectations/`` folder.  This guide will help you configure Great Expectations to store them in an Amazon S3 bucket.
 
-    "Whether I shall turn out to be the hero of my own life, or whether that station will be held by anybody else, these pages must show."
+.. admonition:: Prerequisites: This how-to guide assumes that you have already:
 
+    - Configured a :ref:`Data Context <tutorials__getting_started__initialize_a_data_context>`.
+    - Configured an :ref:`Expectations Suite <tutorials__getting_started__create_your_first_expectations>`.
+    - Configured `boto3 <https://github.com/boto/boto3>`_ to connect to the Amazon S3 bucket where Expectations will be stored.
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
+Steps
+-----
+1. Look for the following lines in the ``great_expectations.yml`` file.
 
-If it would be useful to you, please comment with a +1 and feel free to add any suggestions or questions below.
+    .. code-block:: yaml
+
+        expectations_store_name: expectations_store
+
+        stores:
+            expectations_store:
+                class_name: ExpectationsStore
+                store_backend:
+                    class_name: TupleFilesystemStoreBackend
+                    base_directory: expectations/
+
+The configuration file tells Great Expectations to look for Expectations in a store called ``expectations_store``. The ``base_directory`` for ``expectations_store`` is set to ``expectations/`` by default.
```

Comment:
I'm not sure if this step is necessary, but I feel it makes the story flow more smoothly. Please let me know what you think

Reply:
Yeah, how much of this kind of exposition to include is always a judgement call. This feels good to me.